doc/manuals/admin/perf_tuning.md

```diff
@@ -144,14 +144,41 @@ On the other hand, setting thread pools is a way of "capping" throughput. If the
 all the time, at the end the queue saturates and you will end up losing incoming request or ongoing
 notifications.
 
-Finally, you may have thread exhaustion problems if you don’t use thread pools. You can detect that
+Finally, you may have *thread exhaustion problems* if you don't use thread pools. You can detect that
 situation by two symptoms. First, an unexpectedly high number of threads associated to the process.
 Second, error messages like this appearing in the logs:
 
 ```
 Runtime Error (error creating thread: ...)
 ```
 
+On the other hand, you may have *file descriptors exahustion problems* if your pools are not correctly
+sized. In particular, the following inequity ensures the number of file descriptors used by Orion
+is below the operating system limit:
+
+```
+max fds > 5 * n + max cons + db pool size + extra
+```
+
+where
+
+* **max fds** is the per process file descriptors limit, i.e. the output of the `ulimit -n` command. It can
+  be changed with `ulimit -n <new limit>`.
+* **n**, number of threads in the notification threads pool size. The 5 factor is due to each thread can
```

Comment:
number of threads in the notification thread pool

  or

the size of the notification thread pool